source/userguide/active_learning.rst

```diff
@@ -0,0 +1,226 @@
+| Least Confidence Sampling | From the available queries in the batch, this sampling strategy samples queries with the      |
+|                           | lowest max confidence score across any class, i.e., queries that are the least confident about|
+|                           | any of the available classes in the classifier.                                               |
++---------------------------+-----------------------------------------------------------------------------------------------+
+| Entropy Sampling          | Calculates the entropy score of the classifier confidences per query. Samples the ones with   |
+|                           | highest entropy.                                                                              |
++---------------------------+-----------------------------------------------------------------------------------------------+
+| Margin Sampling           | From the available queries in the batch, this sampling strategy samples queries with the      |
+|                           | that have the lowest confidence score difference between the top two class confidence scores  |
+|                           | for the query. This difference is referred to as the "margin".                                |
++---------------------------+-----------------------------------------------------------------------------------------------+
+| Disagreement Sampling     | Across n runs of the classifier, this sampling strategy calculates an agreement score for     |
+|                           | every query (% of classifiers that voted for the most frequent class). The queries are then   |
+|                           | ranked from lowest classifier agreement to highest and the sampled in order.                  |
++---------------------------+-----------------------------------------------------------------------------------------------+
+| KL Divergence Sampling    | Across n runs of the classifier, this sampling strategy calculates the KL divergence between  |
+|                           | average confidence distribution across all classifiers for a given class and the confidence   |
+|                           | distribution for a given query for that class. Queries with higher divergence are sampled.    |
++---------------------------+-----------------------------------------------------------------------------------------------+
+| Ensemble Sampling         | Combines ranks from all the above heuristics and samples in order.                            |
++---------------------------+-----------------------------------------------------------------------------------------------+
+
+Tuning Levels
+^^^^^^^^^^^^^
+
+Tuning level defines the level at which the classifier evaluation is done and for which the confidence scores or model probabilites are passed onto the strategies defined above. One of three strategies can be employed - domain, intent or joint.
```

Comment:
```suggestion
Since MindMeld defines a hierarchy of domains and intents, the various heuristics can be computed by using the confidence scores or probabilities of either the domain or intent classifiers or both. This level  is indicated by the Tuning level in the config. 
```